frontend/public/components/storage/create-pvc.tsx

```diff
@@ -256,9 +256,9 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
         {useSelector && (
           <NameValueEditorComponent
             nameValuePairs={nameValuePairs}
-            valueString="Selector"
-            nameString="Label"
-            addString="Add Value"
+            valueString={t('persistent-volume-claim~Selector')}
+            nameString={t('public~Label')}
+            addString={t('persistent-volume-claim~Add Value')}
```

Comment:
Not new, but we're trying to fix the case to conform to PatternFly standards as well internationalize the console.

```suggestion
            addString={t('persistent-volume-claim~Add value')}
```

Reply:
updated